crates/aptos-logger/src/macros.rs

```diff
@@ -12,8 +12,6 @@ macro_rules! log {
             $level,
             env!("CARGO_CRATE_NAME"),
             module_path!(),
-            file!(),
-            line!(),
             concat!(file!(), ':', line!()),
```

Comment:
why?

Reply:
because we don't need `file` and `line` as separate struct fields imo. the `location` (renamed now to `source.file`) is already a superset of this. Removing this redundancy is good for readability of the json logs and also reduces the logging volume a bit.

Reply:
I see... I feel a little disconnected to make that assertion. I mean you upped the bar to searching to some extent ;)

Reply:
yeah I initially thought so as well, but after some thought I concluded:
- nobody searches by line numbers alone (at least not without the file context)
- substring searches on file name will still be possible easily, just not exact-match filtering on a file path anymore.
- If I want to do some log analytics (for example to find the biggest offenders in terms of log volume), I think it much more likely group by `file_path:line` than file name alone.

Reply:
This was something I questioned in the original design (it was previously location and then split)